apps/services/license-api/src/app/modules/license/test/license.service.spec.ts

```diff
@@ -47,11 +47,14 @@ const cacheStore = new Map<string, unknown>()
 const licenseIds = Object.values(LicenseId)
 
 const createCacheData = (licenseId: LicenseId): BarcodeData<LicenseType> => ({
-  nationalId: faker.datatype.number({ min: 10, max: 10 }).toString(),
+  // Question:
+  // Was this meant to always generate the value "10"?
+  // It seems like you want a numeric string with the length of 10 here instead.
+  nationalId: faker.number.int({ min: 10, max: 10 }).toString(),
```

Comment:
Mentioning this code as a comment so the maintainers might be able to give more context.
What was intend to generate here? Based on the property name this should propably use one of the national id generation function in your codebase (?).